loop-templates/templates/package.json.squirrelly

```diff
@@ -95,6 +95,9 @@
 {{ @each(it.aptitudes) => aptitude }}
 {{ @if (aptitude === 'clipboard') }}
       "clipboard": {},
+{{ #elif (aptitude === 'cursor' && !it.aptitudes.includes('keyboard')) }}
```

Comment:
I think instead of the AND check here, you can put an OR check on the keyboard aptitude. Otherwise if both cursor and keyboard are selected, you actually don't get cursor.

So here would be:
`{{ #elif (aptitude === 'cursor' ) }}`

and down on L105 would be
`{{ #elif (aptitude === 'keyboard' || aptitude === 'cursor') }}`
or if that causes an issue with the looping you could be safer with
`{{ #elif (aptitude === 'keyboard' || (aptitude === 'cursor' && !it.aptitudes.includes('keyboard'))) }}`